six/six.cpp

```diff
@@ -25,13 +25,20 @@ const char *Six::getExtensionModuleName(six_module_t m) {
     }
 }
 
-const char *Six::getUnknownModuleName() { return datadog_agent_six_unknown; }
+const char *Six::getUnknownModuleName() { 
```

Comment:
these should go on a dedicated PR along with required changes to `.clang-format` file since coding style is enforced on the CI

Reply:
I did the change to `.clang-format` in this PR, too.